packages/notifi-react-card/lib/components/subscription/NotifiSubscriptionCardContainer.tsx

```diff
@@ -23,15 +23,24 @@ export const NotifiSubscriptionCardContainer: React.FC<
   loadingRingColor,
   loadingSpinnerSize,
   onClose,
+  demoByJSONConfig,
 }: React.PropsWithChildren<NotifiSubscriptionCardProps>) => {
   const { isInitialized } = useNotifiSubscribe({ targetGroupName: 'Default' });
-  const { loading } = useNotifiSubscriptionContext();
+  const { loading, setDemoPreview } = useNotifiSubscriptionContext();
+
+  useEffect(() => {
+    demoByJSONConfig ? setDemoPreview(demoByJSONConfig) : setDemoPreview(null);
+  }, [demoByJSONConfig]);
+
   const inputDisabled = loading || !isInitialized;
 
-  const card = useSubscriptionCard({
-    id: cardId,
-    type: 'SUBSCRIPTION_CARD',
-  });
+  const card = useSubscriptionCard(
+    {
+      id: cardId,
+      type: 'SUBSCRIPTION_CARD',
+    },
+    demoByJSONConfig,
```

Comment:
instead of parameterizing useSubscriptionCard this way, I think we can mount a different component (maybe don't mount a NotifiSubscriptionCardContainer?) if we are using demo view.